x-pack/legacy/plugins/uptime/public/components/functional/integration_group.tsx

```diff
@@ -37,13 +37,12 @@ export const IntegrationGroup = ({
   isApmAvailable,
   isInfraAvailable,
   isLogsAvailable,
-  monitor,
-  monitor: { ping },
+  summary,
 }: IntegrationGroupProps) => {
-  const domain = get<string>(ping, 'url.domain', '');
-  const podUid = get<string | undefined>(ping, 'kubernetes.pod.uid', undefined);
-  const containerId = get<string | undefined>(ping, 'container.id', undefined);
-  const ip = get<string | undefined>(ping, 'monitor.ip');
+  const domain = get<string>(summary, 'state.url.domain', '');
```

Comment:
This seems like it is potentially risky, investigate further with additional tests.

Reply:
What do you mean by risky? All documents should have `url.domain`